lib/mongoose_push/telemetry.ex

```diff
@@ -0,0 +1,36 @@
+defmodule MongoosePush.Telemetry do
+  @moduledoc """
+  Module responsible for handling telemetry events
+  """
+
+  def attach_all do
+    _ =
+      :telemetry.attach_many(
```

Comment:
How about splitting this into two functions so that it scales better when we add more?

Reply:
Actually, how about chaining this module into behaviour that `MongoosePush.Telemetry.APNSHandler` and `MongoosePush.Telemetry.FCMHandler` implement, with two functions: `handle_event/4` and `attach/0`? This way each handler module registers its own handlers.